autoload/purescript/ide.vim

```diff
@@ -42,7 +42,7 @@ fun! purescript#ide#call(input, errorm, isRetry, cb, ...)
     let cwdcommand = {'command': 'cwd'}
 
     let jobid = purescript#job#start(
-	  \ ["purs", "ide", "client", "-p", g:psc_ide_server_port],
+	  \ g:psc_ide_server_runner + ["ide", "client", "-p", g:psc_ide_server_port],
```

Comment:
A buffer local variable might be a better choice, it will allow to use one com session to work with different projects.

Reply:
If we are at this, maybe we should also redeclare `psc_ide_server_port`as buffer local variable, with the global variable as it's default value to avoid breaking changes.